stacks/towerofhanoi.test.js

```diff
@@ -0,0 +1,49 @@
+const TowerOfHanoi = require('./TowerOfHanoi');
+
+test('Number of discs on tower A should be 4 initially', () => {
```

Comment:
none of the tests are able to determine the order. I think we can pop elements from stack c one by one to assert that order is correct